### ISSUE TITLE
Feiv : More diagnostic tests on the first stage regression need to be added

### ISSUE THREAD

**Issue by Jayhyung** [py-econometrics/pyfixest#543]
Related to issue #382, more diagnostic tests on the first stage regression of 2SLS need to be added. 

More details will come after the PR #536 is merged!

**Comment by s3alfisc** [comment]
PR #536 is merged =) So the next step would be to always run an F-Test for the first stage IV?

**Comment by Jayhyung** [comment]
Yes, this will be the very first step to do. I'm working on it, will implement the unit test against @apoorvalal's ivDiag package. Stay tuned!
